docs/strategy-advanced.md

```diff
@@ -398,7 +398,10 @@ class AwesomeStrategy(IStrategy):
 ```
 
 !!! Warning
-    Modifying entry and exit prices will only work for limit orders. Depending on the price chosen, this can result in a lot of unfilled orders.
+    Modifying entry and exit prices will only work for limit orders. Depending on the price chosen, this can result in a lot of unfilled orders. By default the maximum allowed distance between the current price and the custom price is 2%, this value can be changed in config with the `custom_price_max_distance_percent` parameter.
+
+_Exemple_
+If the new_entryprice is 97, the proposed_rate is 100 and the `custom_price_max_distance_percent` is set to 2%, The retained valid custom entry price will be 98.
```

Comment:
I don't think this will render well, and i think we should prefer an "example box" instead, which will render better in the documentation.

```suggestion
!!! Exemple 
    If the new_entryprice is 97, the proposed_rate is 100 and the `custom_price_max_distance_percent` is set to 2%, The retained valid custom entry price will be 98.
```

Reply:
Can you give me more details or an exemple of what you mean by an "example box"?

Reply:
Ok I understood, i watched on my phone first (without the suggestion)

Reply:
phone versions can be ... dangeorus sometimes :laughing: 

be careful with pressing the "commit" button though - as the parameter name might change (see other commit)...